### PR TITLE
(Testmerge Ready) New Antagonist: Tiger Cooperative Fanatic

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
This antagonist is outlined here: https://hackmd.io/eNu8pH_aRZuuoX04zqEqTQ
This antagonist is not in its final stage of development.  What is currently presented is a functioning prototype with all of the necessary core mechanics in order to create what should be a fun antagonist that adds something new to the game. I'd like to do some testmerges for some feedback before this is shipped. This has been bugtested pretty rigorously, but its possible some unintended interactions still exist.
There were a few fixes made along the way.
Special thanks to @YesterdaysPromise for the sprites for the clothing.
## Why It's Good For The Game
It's a fun new antag with a unquie concept. Fleshes out the setting further.
## Changelog
:cl: Code from itseasytosee Sprites by YesrerdaysPromise
add: A new antagonist has been added. A fanatic stalks maintenance, he has come here in search of his changeling gods, desperate to serve.
/:cl:

### CODE REVIEW

**Comment by Vincent983** [comment]
I know it's a big incentive to absorb the fanatic, but 5-10 DNA points from absorbing a much weaker ally than another changeling who only gives you 2 DNA points seems a bit excessive (though increasing the DNA point gain from absorbing other lings would be neat considering 2 DNA points is literally only 1 or 2 powers)

**Comment by itseasytosee** [comment]
> I know it's a big incentive to absorb the fanatic, but 5-10 DNA points from absorbing a much weaker ally than another changeling who only gives you 2 DNA points seems a bit excessive (though increasing the DNA point gain from absorbing other lings would be neat considering 2 DNA points is literally only 1 or 2 powers)

From reading the code, it looks like absorbing a ling gives you have of their DNA points, which is usually 5 total. Although I agree that aspect needs to be considered.

**Comment by itseasytosee** [comment]
I'm not exactly sure why AntagInfoTigerFanatic.tsx is getting a warning from the linters. The feedback isn't clear

**Comment by MrMelbert** [comment]
> Changelings are once again able to absorb people they have previously DNA stung.

Not a fix, would be a balance change.

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	hardcore_random_bonus = TRUE
+	ui_name = "AntagInfoTigerFanatic"
+	var/blessings = 0
+
+/datum/antagonist/tiger_fanatic/greet()
+	. = ..()
+	to_chat(owner.current, span_changeling("You worship the changeling hive!."))
+	to_chat(owner.current, span_changeling("You have detected the pressence of the changeling hive mind, and have smuggled yourself aboard the station."))
+	to_chat(owner.current, span_changeling("This is your one opertunity to make contact with a changeling, you must be assimilated into the hive in order to accend."))
+	to_chat(owner.current, span_changeling("You have a weak connection to the changeling hivemind, your body has been conditioned to make you a perfect offering to the changelings."))
```
Comment (MrMelbert) [inline code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:18]: This should all be in the antag info panel

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	owner.announce_objectives()
+
+/datum/antagonist/tiger_fanatic/on_gain()
+	owner.special_role = "tiger fanatic"
```
Comment (MrMelbert) [inline code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:22]:
```suggestion
	owner.special_role = ROLE_TIGER_FANATIC
```

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	forge_objectives()
+	. = ..()
+
+/datum/antagonist/tiger_fanatic/forge_objectives()
+	var/datum/objective/be_absorbed/absorbed_escape/absorbed = new
+	var/datum/objective/changeling_blessed/blessed = new
+	absorbed.owner = owner
+	blessed.owner = owner
+	objectives += absorbed
+	objectives +=blessed
```
Comment (MrMelbert) [inline code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:33]:
```suggestion
	objectives += blessed
```

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+/datum/objective/changeling_blessed
+	name = "be blessed by a changeling"
+	explanation_text = "Have a changeling use their powers on you 3 times."
+	martyr_compatible = TRUE
+	admin_grantable = TRUE
+	completed = FALSE
+	var/blessings_required = 3
+
+/datum/objective/changeling_blessed/update_explanation_text()
+
```
Comment (MrMelbert) [inline code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:76]: Missing something here

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+/datum/objective/changeling_blessed/check_completion()
+	var/datum/antagonist/tiger_fanatic/tiger_fanatic = owner.has_antag_datum(/datum/antagonist/tiger_fanatic)
+	if(!tiger_fanatic)
```
Comment (MrMelbert) [inline code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:80]:
```suggestion
	if(isnull(tiger_fanatic))
```

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+		return FALSE
+	if(tiger_fanatic.blessings >= blessings_required)
+		return TRUE
+	return FALSE
+
+/proc/create_tiger_fanatic(spawn_loc)
```
Comment (MrMelbert) [inline code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:86]: Does this need to be a global proc?

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/tiger_fanatic/items.dm [hunk 7]
```diff
@@ -0,0 +1,77 @@
+/obj/item/clothing/suit/hooded/tiger_co_hoodie
+	name = "religious hoodie"
+	desc = "A red hoodie with a orange cross mark going through its back and front. Or an double axe. Or an 'T', perhaps?"
```
Comment (Fikou) [inline code/modules/antagonists/tiger_fanatic/items.dm:3]:
```suggestion
	desc = "A red hoodie with a orange cross mark going through its back and front. Or an double axe. Or a 'T', perhaps?"
```

File: code/modules/antagonists/tiger_fanatic/items.dm [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+	body_parts_covered = CHEST|GROIN|ARMS
+	cold_protection = CHEST|GROIN|ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	inhand_icon_state = null
+	species_exception = list(/datum/species/golem)
+	hoodtype = /obj/item/clothing/head/hooded/tiger_co_hood
+
+/obj/item/clothing/head/hooded/tiger_co_hood
+	name = "red hood"
+	desc = "A red hood with orange trim. Not exacly the type for bringing things to grandma in. Nor for hunting wolves."
```
Comment (Fikou) [inline code/modules/antagonists/tiger_fanatic/items.dm:27]:
```suggestion
	desc = "A red hood with an orange trim. Not exactly the type for bringing things to grandma in. Nor for hunting wolves."
```

File: code/modules/antagonists/tiger_fanatic/items.dm [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+	icon = 'icons/obj/clothing/head/chaplain.dmi'
+	worn_icon = 'icons/mob/clothing/head/chaplain.dmi'
+	icon_state = "tiger_co_hood"
+	body_parts_covered = HEAD
+	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
+	inhand_icon_state = null
+
+/obj/item/clothing/suit/hooded/tiger_co_inquisitor
+	name = "Tiger Co Inquisitor's Armour"
```
Comment (Fikou) [inline code/modules/antagonists/tiger_fanatic/items.dm:36]:
```suggestion
	name = "tiger inquisitor's armor"
```

File: code/modules/antagonists/tiger_fanatic/items.dm [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+	desc = "A proper attire of Tiger Co associated chaplain-inquisitors. Almost suprising it doesn't feature a fleshy third arm or something."
```
Comment (Fikou) [inline code/modules/antagonists/tiger_fanatic/items.dm:37]:
```suggestion
	desc = "The proper attire of Tiger Cooperative associated chaplain-inquisitors. Almost surprising it doesn't feature a fleshy third arm or something."
```

File: code/modules/antagonists/tiger_fanatic/items.dm [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+	laser = 25
+	energy = 35
+	bomb = 25
+	fire = 50
+	acid = 50
+	wound = 10
+
+/obj/item/clothing/under/syndicate/tacticool/puffed_shirt
+	name = "red puffed shirt"
+	desc = "Mix of streetwear and utility. Though it might look expensive, its mostly made from recycled packages of Waffle Corp. products."
```
Comment (Fikou) [inline code/modules/antagonists/tiger_fanatic/items.dm:59]:
```suggestion
	desc = "Mix of streetwear and utility. Though it might look expensive, it's mostly made from recycled packages of Waffle Corp. products."
```

File: code/modules/antagonists/tiger_fanatic/items.dm [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+	icon_state = "puffed_shirt"
+
+/obj/item/clothing/mask/mummy/tiger_co
+	name = "set of bandadges"
+	desc = "To cover results of ritual scarification, disfigurement from drugs, and your ugly face."
+
+/obj/item/clothing/gloves/bandages
+	name = "set of arm wraps"
+	desc = "These were used to cover up wounds along the arms. Deep ones."
```
Comment (Fikou) [inline code/modules/antagonists/tiger_fanatic/items.dm:68]:
```suggestion
	desc = "These are used to cover up wounds along the arms. Deep ones."
```

**Comment by DaydreamIQ** [comment]
Being able to get 50-100% more points and DOUBLE that for absorbing a normal human seems like overkill. With other lings there's the fact that it's incredibly difficult to keep a ling down long enough for you to absorb without them reviving and escaping, but a person who's WILLING to be absorbed to massively increase your power is absurd

**Comment by D4C-420** [comment]
> Cryo-sting will also add 30 units of cryoxadone in addition to the frost oil, which will lead to the healing of the tiger. 

cyroaxadone requires the subject to be sleeping in-order to heal. Could throw in a sleeping chem for it to work.

**Comment by tralezab** [comment]
I'd rather launch Tiger Cooperative testing some strong balance and tweaking downwards instead of launching him a lot weaker and tweaking upwards because Antagonists in general already get rolled by the station

**Comment by the-orange-cow** [comment]
Idea:  a changeling has the capability to turn a tigercoop into a fallen changeling at the expense of some of their points.  Could be a reward if the tigercoop completes certain objectives?  Might be too OP, but thought I'd throw the thought out there.

**Comment by itseasytosee** [comment]
> > Changelings are once again able to absorb people they have previously DNA stung.
> 
> Not a fix, would be a balance change.

You can still currently do it, you just have to sting a bunch of people and then drop the disguise.  I'm fairly certain this interaction is unintentional.

File: code/modules/antagonists/changeling/powers/absorb.dm [hunk 13]
```diff
@@ -132,6 +137,46 @@
...
+		var/original_name = target.real_name
+
+		target.body_position_pixel_y_offset = old_body_position
+		target.set_lying_angle(90)
+
+		var/mob/camera/imaginary_friend/changeling_echo/worshiper = target.change_mob_type(
+			new_type = /mob/camera/imaginary_friend/changeling_echo,
+			location = get_turf(target),
+			delete_old_mob = FALSE,
+		)
```
[the remark this answers is not in the document]
Comment (itseasytosee) [reply code/modules/antagonists/changeling/powers/absorb.dm:175]: It ghostizes them first. So they do leave a husk.

File: code/datums/brain_damage/imaginary_friend.dm [hunk 14]
```diff
@@ -532,3 +532,41 @@
 	real_name = "[owner.real_name]?"
 	name = real_name
 	human_image = icon('icons/mob/simple/lavaland/lavaland_monsters.dmi', icon_state = "curseblob")
+
+/datum/brain_trauma/special/imaginary_friend/trapped_owner/changeling_echo
```
[the remark this answers is not in the document]
Comment (itseasytosee) [reply code/datums/brain_damage/imaginary_friend.dm:536]: Its a remnant from an older implementation I forgot to clean up.

File: code/controllers/subsystem/dynamic/dynamic_rulesets_midround.dm [hunk 15]
```diff
@@ -861,6 +861,55 @@
...
+	return ..()
+
+/datum/dynamic_ruleset/midround/from_ghosts/tiger_fanatic/ready(forced = FALSE)
+	var/list/changelings = list()
+	if(forced)
+		return ..()
+
+	for (var/mob/creature in living_antags)
+		if(IS_CHANGELING(creature))
+			changelings += creature
```
[the remark this answers is not in the document]
Comment (itseasytosee) [reply code/controllers/subsystem/dynamic/dynamic_rulesets_midround.dm:891]: good shout.

File: code/modules/antagonists/changeling/powers/mutations.dm [hunk 16]
```diff
@@ -214,6 +214,9 @@
 	. = ..()
 	if(!proximity)
 		return
+	if(fake)
+		return
```
[the remark this answers is not in the document]
Comment (itseasytosee) [reply code/modules/antagonists/changeling/powers/mutations.dm:218]: alright.

**Comment by itseasytosee** [comment]
> > Cryo-sting will also add 30 units of cryoxadone in addition to the frost oil, which will lead to the healing of the tiger.
> 
> cyroaxadone requires the subject to be sleeping in-order to heal. Could throw in a sleeping chem for it to work.

this got replaced with a different healing chemical for that exact reason.

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm (hunk 6 above)
Comment (itseasytosee) [reply code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:86]: No, but there is precedent for it being one. Ninja does the same thing. I don't know where else you would stash it, because the body is created before the datum is applied.

**Comment by itseasytosee** [comment]
> I'd rather launch Tiger Cooperative testing some strong balance and tweaking downwards instead of launching him a lot weaker and tweaking upwards because Antagonists in general already get rolled by the station

Giving them sleeping carb might be a good idea. Otherwise they could get a energy dagger or something of the like. A stim pack would probably be thematic.

**Comment by excalisword** [comment]
Given how the only information about the tiger fanatics combat style is that they use chemicals to fight beyond what a normal crew member can, maybe you could give them refilling chemical implants that they can trigger to pump their body with a limited amount of combat chems like ephedrine or bath salts.

**Comment by Rex9001** [comment]
> > I'd rather launch Tiger Cooperative testing some strong balance and tweaking downwards instead of launching him a lot weaker and tweaking upwards because Antagonists in general already get rolled by the station
> 
> Giving them sleeping carb might be a good idea. Otherwise they could get a energy dagger or something of the like. A stim pack would probably be thematic.

You could give them ritual wine from the flesh meateor ruin

**Comment by TheVekter** [comment]
I like this idea, though I would like to see some kind of benefit for the changeling to keep a Tiger alive if they'd prefer to do so. There's obviously intrinsic benefits to having a teammate, but I'd be interested in something mechanical to prevent most interactions with them being "hey come here so I can eat you".

File: code/modules/antagonists/changeling/powers/tiny_prick.dm [hunk 17]
```diff
@@ -261,6 +274,11 @@
 	..()
 	log_combat(user, target, "stung", "LSD sting")
 	addtimer(CALLBACK(src, PROC_REF(hallucination_time), target), rand(30 SECONDS, 60 SECONDS))
+	if(target.reagents && IS_TIGER_FANATIC(target))
+		var/datum/antagonist/tiger_fanatic/tiger_fanatic = target.mind.has_antag_datum(/datum/antagonist/tiger_fanatic)
+		target.reagents.add_reagent(/datum/reagent/medicine/changelingadrenaline, 4)
+		target.reagents.add_reagent(/datum/reagent/medicine/changelingadrenaline, 3)
```
[the remark this answers is not in the document]
Comment (itseasytosee) [reply code/modules/antagonists/changeling/powers/tiny_prick.dm:280]: was supposed to be haste.

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm (hunk 1 above)
Comment (itseasytosee) [reply code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:18]: You do think it should also be removed from the greet()?

**Comment by itseasytosee** [comment]
They have received a buff to their starting gear. They start with a smuggler's satchel in hand which contains:
A combat knife (20 damage)
A desert eagle (7 shots, 60 damage)
A spare magazine
A Bio terror foam grenade
A doorjack
A flash full of ritual wine (which heals them)

**Comment by the-orange-cow** [comment]
> I like this idea, though I would like to see some kind of benefit for the changeling to keep a Tiger alive if they'd prefer to do so. There's obviously intrinsic benefits to having a teammate, but I'd be interested in something mechanical to prevent most interactions with them being "hey come here so I can eat you".

Having a loyal helper can be supremely useful in and of itself, but knowing the playerbase something like you describe would certainly help drive it home.

**Comment by tralezab** [comment]
> I like this idea, though I would like to see some kind of benefit for the changeling to keep a Tiger alive if they'd prefer to do so. There's obviously intrinsic benefits to having a teammate, but I'd be interested in something mechanical to prevent most interactions with them being "hey come here so I can eat you".

I agree. We should always be looking to incentivize playing in interesting ways!

**Comment by MrMelbert** [comment]
> You can still currently do it, you just have to sting a bunch of people and then drop the disguise. I'm fairly certain this interaction is unintentional.

I'd disagree because it makes sense to me, so changelings have to actually think about DNA stinging / manager their DNA rather than it being a no-brainer.

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm (hunk 6 above)
Comment (MrMelbert) [reply code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:83]: Well ninja does it because it's like over a decade old, and it used to be in the VV menu.

 I don't think this needs to be global - You only invoke it one place, on the dynamic ruleset, so it can just be `/datum/dynamic_ruleset/midround/from_ghosts/tiger_fanatic/proc/create_tiger_fanatic()`

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm (hunk 1 above)
Comment (MrMelbert) [reply code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:18]: Yes, antag info panel is the replacement for greet

File: code/modules/reagents/chemistry/reagents/drinks/alcohol_reagents.dm [hunk 18]
```diff
@@ -2448,6 +2448,20 @@
 	var/atom/movable/plane_master_controller/game_plane_master_controller = psychonaut.hud_used.plane_master_controllers[PLANE_MASTERS_GAME]
 	game_plane_master_controller.add_filter("ritual_wine", 1, list("type" = "wave", "size" = 1, "x" = 5, "y" = 0, "flags" = WAVE_SIDEWAYS))
 
+/datum/reagent/consumable/ethanol/ritual_wine/on_mob_metabolize(mob/living/carbon/psychonaut, seconds_per_tick, times_fired)
+	. = ..()
+	if(IS_TIGER_FANATIC(psychonaut))
```
Comment (MrMelbert) [inline code/modules/reagents/chemistry/reagents/drinks/alcohol_reagents.dm:2453]: Early return

**Comment by itseasytosee** [comment]
> > You can still currently do it, you just have to sting a bunch of people and then drop the disguise. I'm fairly certain this interaction is unintentional.
> 
> I'd disagree because it makes sense to me, so changelings have to actually think about DNA stinging / manage their DNA rather than it being a no-brainer.

Changelings can now no longer absorb the DNA from a tiger cooperative via a sting.

File: tgui/packages/tgui/interfaces/AntagInfoTigerFanatic.tsx [hunk 19]
```diff
@@ -0,0 +1,115 @@
...
+    <Section fill title="Abilities">
+      <Stack fill>
+        <Stack.Item basis={0} grow>
+          <Stack fill vertical>
+            <Stack.Item basis={0} textColor="label" grow>
+              Through many rituals, psychedelic induced comas, and scarification
+              you have managed to forge a weak connection to the
+              <span style={hivemindstyle}>&ensp;changeling hivemind. </span>
+              You can speak over the hivemind by using
+              <span style={absorbstyle}>&ensp;:g</span>. Contact the
```
Comment (MrMelbert) [inline tgui/packages/tgui/interfaces/AntagInfoTigerFanatic.tsx:96]: Also this radio key is technically not hardcoded so you should pass it via ui data. 

Also also you should mention using `.[key]` or `:[key]` rather than one or the other.

Comment (itseasytosee) [reply tgui/packages/tgui/interfaces/AntagInfoTigerFanatic.tsx]: competed

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm (hunk 6 above)
Comment (itseasytosee) [reply code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm:89]: works for me

File: code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm (hunk 1 above)
Comment (itseasytosee) [reply code/modules/antagonists/tiger_fanatic/tiger_fanatic.dm]: done

**Comment by SomeoneYouProbablyKnow** [comment]
If i'm reading right you are giving them access to ling comms? I feel like that makes the first ling - fanatic interaction real anticlimactic. You just :g them, meet up and they absorb you or you team up.

What if instead you gave the fanatic a ling radar ability. That way the fanatic has to find their ling, approach them. That's a fun interaction i think. It can even be subverted by clever crewmen, they can pretend to be a fanatic to get the ling to reveal themselves.

Additionally it can serve as a potential reason to keep the fantic alive as you kinda get a free ability. (More so if the extra dna for absorbtion is not so absurdly high)

**Comment by xzero314** [comment]
If I was a changeling that had gone to great lengths to infiltrate undetected, I would be super annoyed at a group of people bringing up the existence of changelings. 

These sound like people that changelings will want to kill and be rid of ASAP before they give away the idea of lings being on station. 

Basically what I am saying is these people showing up only when changelings are around and making a big deal out of loving changelings would be counter to their purpose of being beneficial to changelings since it would announce that there are lings around.

File: code/datums/brain_damage/imaginary_friend.dm [hunk 20]
```diff
@@ -532,3 +532,20 @@
...
+
+
+/mob/camera/imaginary_friend/changeling_echo/setup_appearance(original_appearance, original_name)
+	human_image = original_appearance
+	real_name = original_name
+	name = original_name
+	Show()
+
+/mob/camera/imaginary_friend/changeling_echo/greet()
+	to_chat(src, span_changeling(span_bold("You are absorbed, but you your mind persist! You you have become one with the hive!")))
```
Comment (necromanceranne) [inline code/datums/brain_damage/imaginary_friend.dm:550]:
```suggestion
	to_chat(src, span_changeling(span_bold("You are absorbed, but you your mind persists! You you have become one with the hive!")))
```

**Comment by itseasytosee** [comment]
Bonuses from absorbing a tiger fan have changed
DNA points: 5-10 -> 5
Chemicals: 75 - 125  -> 35

**Comment by ATH1909** [comment]
> If I was a changeling that had gone to great lengths to infiltrate undetected, I would be super annoyed at a group of people bringing up the existence of changelings.
> 
> These sound like people that changelings will want to kill and be rid of ASAP before they give away the idea of lings being on station.
> 
> Basically what I am saying is these people showing up only when changelings are around and making a big deal out of loving changelings would be counter to their purpose of being beneficial to changelings since it would announce that there are lings around.

image

**Comment by itseasytosee** [comment]
changes the deagle to a markov because people have suggested to me that the deagle isn't even in close to the realm of balanced. fair enough

**Comment by itseasytosee** [comment]
I think its in a position where if we start trying it out, we can see what works, and where we might need to go back to the drawing board (TM ready)

**Comment by Admiral-Gigaton** [comment]
> Idea: a changeling has the capability to turn a tigercoop into a fallen changeling at the expense of some of their points. Could be a reward if the tigercoop completes certain objectives? Might be too OP, but thought I'd throw the thought out there.

what does a fallen changeling have? does it have powers, anything? can it regain powers? all i know is that its an antag tag that you apply to someone who was a ling but got got, and are now pretty much just humans with a valid card

**Comment by Vincent983** [comment]
Yeah I'm pretty sure fallen changelings don't have any powers (unless we give them some).

**Review by Mothblocks — CHANGES_REQUESTED** [review]
this cannot have the name "tiger cooperative fanatic" that doesnt mean anything to anyone and is too wordy. work with maintainers and others in coding general to figureo ut an actual name. this includes the codebase itself lest we get another "heretic is actually eldritch cultist" situation

**Comment by gorillion** [comment]
this is thematically a cool antagonist, but like, are changelings already not the most powerful solo antagonists in the game? i really don't see how giving them a specialized helper role is going to be a net positive for the game at all. maybe if this is something changelings could brainwash/convert people into, it would be more interactive and interesting, but as is it doesn't seem fun for anyone who isn't a changeling.

**Comment by itseasytosee** [comment]
> this cannot have the name "tiger cooperative fanatic" that doesnt mean anything to anyone and is too wordy. work with maintainers and others in coding general to figureo ut an actual name. this includes the codebase itself lest we get another "heretic is actually eldritch cultist" situation

Your concern has been addressed.

**Comment by itseasytosee** [comment]
This dies for now. But it might live again. Someday.
